test/correctness/chunk.cpp

```diff
@@ -28,8 +28,22 @@ int main(int argc, char **argv) {
     for (int i = 0; i < 32; i++) {
         for (int j = 0; j < 32; j++) {
```

Comment:
This is one of the most basic tests - it checks if compute_at works. I'd personally prefer switching the test to integers than have it also test D3D math precision.